src/ray/gcs/gcs_server/gcs_server.cc

```diff
@@ -54,6 +54,9 @@ GcsServer::GcsServer(const ray::gcs::GcsServerConfig &config,
     : config_(config),
       storage_type_(GetStorageType()),
       main_service_(main_service),
+      pubsub_io_context_("pubsub_io_context"),
+      task_io_context_("task_io_context"),
```

Comment:
I think it's better for GcsTaskManager to own and manage its io_context like before?

Reply:
why do we need that? I think all managers can share a same style.